.travis.yml

```diff
@@ -32,6 +32,8 @@ matrix:
           env: SYMFONY_VERSION=3.0.x
         - php: 7.0
           env: SYMFONY_VERSION=3.1.x
+        - php: 7.1
+          env: SYMFONY_VERSION=4.0.x
```

Comment:
For this version, the minimum stability needs to be enforced to be beta as there is no stable release yet. You can best do that probably by also defining a stability env var here like this:

```yaml
- php: 7.1
  env: SYMFONY_VERSION=4.0.x stability=beta
```

And then enforce this before the installation using the `composer config` command like this:

```yaml
install:
  - if [ "$stability" != "" ]; then composer config minimum-stability $stability; endif
  - if [ "$dependencies" = "lowest" ]; then composer update --prefer-lowest --prefer-stable --prefer-dist --no-interaction; else composer update --prefer-dist --no-interaction; fi;
```